mobile/src/main/res/values-de/strings.xml

```diff
@@ -15,78 +15,78 @@
     <string name="app_discovery">Starte Suche</string>
     <!-- Main menu items -->
     <string name="mainmenu_openhab_preferences">Einstellungen</string>
-    <string name="mainmenu_openhab_selectsitemap">Wähle Sitemap</string>
-    <string name="mainmenu_openhab_writetag">Schreibe NFC Tag</string>
-    <string name="mainmenu_openhab_info">openHAB Info</string>
-    <string name="mainmenu_openhab_clearcache">Lösche Bildercache</string>
+    <string name="mainmenu_openhab_selectsitemap">Sitemap-Auswahl</string>
```

Comment:
This wasn't present before either, though (see old line 18). Maybe 'Standard-Sitemap auswählen'?